src/server/server.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+const resourceRouter = require('./routes/resourceRouter');
+const PORT = 3000;
+
+// parse request body
+app.use(express.json());
+app.use(express.urlencoded({ extended: true}));
+
+// set up routers
+app.use('/resource', resourceRouter);
+
+
+// send main app
+app.get('/', res.status(200).sendFile(path.resolve(__dirname, './client/index.html')));
```

Comment:
add anonymous function to trigger sending back of response, and remember to return, please! :)

Reply:
done